src/elements/content-uploader/ItemList.scss

```diff
@@ -5,3 +5,7 @@
     border-bottom: 1px solid $lighter-grey-blue;
     outline: none;
 }
+
+.bcu-item-list-action-column:last-child {
```

Comment:
What is the `:last-child` in this case? Can you attach a screenshot of the new experience?

Reply:
With the new UI the `:last-child` would be the remove button that is newly separated from the in-progress/retry/completed icon.
<img width="1680" alt="Screen Shot 2019-08-30 at 9 48 14 AM" src="https://user-images.githubusercontent.com/29547218/64037965-c31c4780-cb0b-11e9-8088-e0083d210222.png">

Reply:
The cell spacing, content alignment, and icon sizes look a bit off in this screenshot. Each cell, including those in header, should follow the same vertical lines. I think we may get a better result if we make each action cell a certain size, center its icon, and apply a consistent height/width/margin-left.

Reply:
<img width="1680" alt="Screen Shot 2019-09-03 at 10 28 16 AM" src="https://user-images.githubusercontent.com/29547218/64195678-ed854200-ce36-11e9-8569-e02564773463.png">
@jstoffan Does this alignment make a bit more sense? I shifted the Resume button in the header over, standardized the size of each icon in the item rows, removed the extra padding to the right of the percentage progress, and slightly shifted the icon in the header for collapsing/expanding the uploads manager to the right to make it more evenly placed with the remove icons below it.

Reply:
Looks much better, @rosepierce! Thanks for taking the time to polish it up so nicely.